test/unit/settings-specs.js

```diff
@@ -49,12 +49,12 @@ describe('settings', () => {
 
     it('should update a plist', async () => {
       let originalData = await read(tmpPlist);
-      originalData[0]['com.apple.locationd.bundle-/System/Library/PrivateFrameworks/Parsec.framework']
+      originalData['com.apple.locationd.bundle-/System/Library/PrivateFrameworks/Parsec.framework']
```

Comment:
i realize this was duplicated before, but maybe this should be a constant string?